llvm/lib/Transforms/Vectorize/VPlanAnalysis.cpp

```diff
@@ -185,6 +185,7 @@ Type *VPTypeAnalysis::inferScalarTypeForRecipe(const VPReplicateRecipe *R) {
   case Instruction::FPToUI:
   case Instruction::PtrToInt:
   case Instruction::IntToPtr:
+  case Instruction::AddrSpaceCast:
```

Comment:
Try to keep lex ordering (arguably partly broken at the moment but move to correct position)